src/main/java/EmptyAction.java

```diff
@@ -0,0 +1,10 @@
+public class EmptyAction implements Action {
```

Comment:
Good to have some javadoc comments at the start of every file. Also, good to create a public static final constant for anything you use such as the strings that are returned. In the event it is used multiple times, easier to edit also